IO/ECUtil.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace LocalAdmin.V2.IO
+{
+    /// <summary>
+    ///     Error codes for parsing.
+    /// </summary>
+    internal enum ErrorType
+    {
+        /// <summary>
+        ///     Wrong port was given for the game.
+        /// </summary>
+        INVALID_PORT_GIVEN,
+        /// <summary>
+        ///     Unsupported game launch platform.
+        /// </summary>
+        UNSUPPORTED_PLATFORM,
+        /// <summary>
+        ///     Game launcher executable not found.
+        /// </summary>
+        ERROR_FILE_NOT_FOUND,
+    }
+
+    /// <summary>
+    ///     Assistant determining the 
+    ///     correct exit error codes for the application.
+    /// </summary>
+    internal static class ECUtil
```

Comment:
Wouldn't just some ifs on Exits be simplier than this?

Reply:
Rightly.
https://github.com/northwood-studios/LocalAdmin-V2/pull/4/commits/70d43527341e981480aec3ab2bb65971cabe4a0f resolves it.